doc/faq.rst

```diff
@@ -0,0 +1,283 @@
+
+   .. image:: images/faq/faq-reset-spyder.png
+      :alt: Spyder reset button in tools
+
+
+.. dropdown:: Q: How do I change Spyder's language?
+
+   Under :guilabel:`General` in Spyder's :guilabel:`Preferences`, go to the :guilabel:`Advanced settings` tab and select your language from the options displayed under :guilabel:`Language`.
+
+   .. image:: images/faq/faq-change-language.png
+      :alt: Spyder change language in preferences.
+
+
+.. dropdown:: Q: How do I use code cells in Spyder?
+
+   To create a cell in Spyder's :doc:`editor`, type ``#%%`` in your script.
+   Each ``#%%`` will make a new cell.
+   To run a cell, press :kbd:`Shift-Enter` (while your cursor is focused on it) or use the :guilabel:`Run current cell` button in Spyder's toolbar.
+
+   .. image:: images/faq/faq-cells.png
+      :alt: Spyder showing cell generation.
+
+
+.. dropdown:: Q: How do I use plugins with Spyder (e.g. Spyder-Notebook, Spyder-Terminal, Spyder-Unittest)?
+
+   Spyder plugins are available in the ``spyder-ide`` conda channel.
+   To install one, type on the command line (or Anaconda Prompt on Windows):
+
+   .. code-block:: bash
+
+      conda install -c spyder-ide <PLUGIN>
+
+   Replace ``<PLUGIN>`` with the name of the plugin you want to use.
+   For more information on a specific plugin, go to the its repository:
+
+   * `spyder-reports`_
+   * `spyder-unittest`_
+   * `spyder-terminal`_
+   * `spyder-notebook`_
+   * `spyder-memory-profiler`_
+   * `spyder-line-profiler`_
+   * `spyder-vim`_
+   * `spyder-autopep8`_
+
+   .. _spyder-reports: https://github.com/spyder-ide/spyder-reports
+   .. _spyder-unittest: https://github.com/spyder-ide/spyder-unittest
+   .. _spyder-terminal: https://github.com/spyder-ide/spyder-terminal
+   .. _spyder-notebook: https://github.com/spyder-ide/spyder-notebook
+   .. _spyder-memory-profiler: https://github.com/spyder-ide/spyder-memory-profiler
+   .. _spyder-line-profiler: https://github.com/spyder-ide/spyder-line-profiler
+   .. _spyder-vim: https://github.com/spyder-ide/spyder-vim
+   .. _spyder-autopep8: https://github.com/spyder-ide/spyder-autopep8
+
+
+.. dropdown:: Q: How do I clear all variables before executing my code?
+
+   Check the option :guilabel:`Remove all variables before execution` in the :guilabel:`Configuration per file...` dialog under the :guilabel:`Run` menu.
+
+   .. image:: images/faq/faq-remove-variables.png
+      :alt: Spyder showing cell generation.
+
+
+.. dropdown:: Q: How do I run my code in a dedicated console or an external system terminal?
+
+   Select the appropriate option in the :guilabel:`Configuration per file...` dialog under the :guilabel:`Run` menu.
+
+   .. image:: images/faq/faq-run-options.png
+      :alt: Spyder showing cell generation.
+
+
+.. dropdown:: Q: How do I change the syntax highlighting theme?
```

Comment:
```suggestion
.. dropdown:: Q: How do I change the syntax highlighting coloring in the editor?
```

* I thought we had agreed on using coloring instead of theme for this question.
* People usually asks for the editor.

Reply:
```suggestion
.. dropdown:: Q: How do I change the syntax highlighting theme in the Editor?
```

I agree with you about adding the Editor.

As I recall, we went back and forth on this point and @juanis2112 initially sided with you, but then ended up agreeing on syntax highlighting theme. Its the [accepted, understood term](https://en.wikipedia.org/wiki/Syntax_highlighting) for this in English, and is what we ourselves call it in the preferences.

Reply:
Ok, no problem.